src/i18next/init-i18next.js

```diff
@@ -0,0 +1,55 @@
+import i18n from "i18next";
+import i18nextXhrBackend from "i18next-xhr-backend";
+import { initReactI18next } from "react-i18next";
+import ICU from "i18next-icu";
+
+(i18n.default || i18n)
+  .use(i18nextXhrBackend)
+  .use(initReactI18next)
+  .use(ICU)
+  .init({
+    backend: {
+      parse: data => data,
+      loadPath: "{{lng}}|{{ns}}",
+      ajax(url, options, callback, data) {
+        const [language, namespace] = url.split("|");
+        if (namespace && namespace !== "translation") {
+          System.import(decodeHtmlEntity(namespace))
```

Comment:
We import the owning ES module for these translations and expect it to export a function called `importTranslation` that will download the needed translations for it.